resources/user-agents/browsers/firefox/firefox-0-x.json

```diff
@@ -81,79 +83,6 @@
           "platforms": [
             "SunOS"
           ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) Gecko* Firebird/#MAJORVER#.#MINORVER#*",
-          "device": "Linux Desktop",
-          "platforms": [
-            "Linux_32", "Linux"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) Gecko* Firebird/#MAJORVER#.#MINORVER#*",
-          "device": "Windows Desktop",
-          "platforms": [
-            "Win2000", "WinXPa_32", "Win98", "Win95", "Vista_32", "Win7_32", "NT4_B"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) Gecko* Firebird/#MAJORVER#.#MINORVER#*",
-          "device": "Macintosh",
-          "platforms": [
-            "OSX"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) Gecko* Firebird/#MAJORVER#.#MINORVER#*",
-          "device": "general Desktop",
-          "platforms": [
-            "SunOS"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) Gecko* BonEcho/#MAJORVER#.#MINORVER#*",
-          "device": "Windows Desktop",
-          "platforms": [
-            "WinXPa_32"
-          ]
-        }
-      ]
-    },
-    {
-      "userAgent": "Firefox #MAJORVER#.#MINORVER# for Android",
```

Comment:
I'm not clear why this section was here. From my research, there was never a 0.x version of Firefox for Android.  https://en.wikipedia.org/wiki/Firefox_for_Android#Release_history

No tests touched this section so I assume it's safe to remove.